src/client/datascience/ipywidgets/webviewIPyWidgetCoordinator.ts

```diff
@@ -62,7 +62,7 @@ export class WebviewIPyWidgetCoordinator implements IInteractiveWindowListener {
         // There should be an instance of the WebviewMessageCoordinator per notebook webview or interactive window. Create
         // the message coordinator as soon as we're sure what notebook we're in.
         this.notebookIdentity = args.resource;
-        this.messageCoordinator = await CommonMessageCoordinator.create(this.notebookIdentity, this.serviceContainer);
+        this.messageCoordinator = CommonMessageCoordinator.create(this.notebookIdentity, this.serviceContainer);
```

Comment:
It seems that the reason for this change was to allow for a specialization of the `postEmitter.event` of the `CommonMessageCoordinator` before the initialization takes place.

To avoid making `initialize` public and relying on end-user cooperation, I would recommend modifying the factory method `create` to take an optional parameter that allows the caller to inject the `postEmitter`.